tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -721,10 +750,11 @@
                     vm.nodata = true;
                     $interval(function () { vm.showSortExp(); }, 2000, 3);
                 } else {
+                    vm.showDocumentAsPinOrUnpin(searchRequest, response);

```

Comment:
Added one reusable method which will be called at several places for pin and unpin functionality